src/f2py_cmake/cmake/UseF2Py.cmake

```diff
@@ -118,6 +118,7 @@ function(f2py_generate_module NAME)
     COMMAND
       "${${_Python}_EXECUTABLE}" -m numpy.f2py
       "${abs_all_files}" ${_file_arg} ${lower} ${F2PY_F2PY_ARGS}
+    COMMAND_EXPAND_LISTS
```

Comment:
Another way to do it would be to drop the quotes, I think, but I think your way is safer for files with spaces in them. I think we could quote F2PY_F2PY_ARGS too, though?

Reply:
I'll try to remember! I'll probably see him next a week from Wednesday.